### PR TITLE
Self-paced 101 and 201 edits

### PR DESCRIPTION
**Pull request by mtspn**
Changes to training lab to set up for self-paced learning. 

A separate tech doc will be created to describe and promote the training, and to feed rocky-chat with this info. 

Merging blocked by interregnum period.

### CODE REVIEW

**Review by w8896699 — APPROVED** [review]
Looks good to me! Thanks!

File: 101-lab/content/00_overview.md [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
 ## Course Format
 
-This course has been designed to be run _at your own pace_. The lab materials have been designed so that you generally follow the happy path for a simple two-tier deployment. There will be many intentional bugs along the way that will have you encounter real-life troubleshooting techniques!
+We offer this course in two modes, 'live training' or 'self-paced'. 
 
-The course length can vary (based on scheduled). It was designed to be run over a period of a few days rather than just a day or two to have time for isolated touchpoints as well as to provide time for discussion.  During that time __touch points__ will be scheduled with the cohort to talk about special topics and to assist in troubleshooting. 
+If you're attending the live training, we'll run through some background information in a 1-day workshop before the lab begins. 
 
-### This course is new!
+For the self-paced training, you can view the slides and recordings of them being presented [here](https://docs.google.com/presentation/d/1VNxqtRRECNd4uzZZslpLsdKYYXN70Gpu-jCLCD1yF9Y/edit?usp=sharinghere).
```
Comment (ShellyXueHan) [inline 101-lab/content/00_overview.md:20]: I can edit the content from this link,  not sure if it's the same for anyone outside of our team? If so, let's switch it to a view only acces

File: 101-lab/content/00_overview.md [hunk 2]
```diff
@@ -1,31 +1,42 @@
...
+If you're attending the live training, we'll run through some background information in a 1-day workshop before the lab begins. 
 
-### This course is new!
+For the self-paced training, you can view the slides and recordings of them being presented [here](https://docs.google.com/presentation/d/1VNxqtRRECNd4uzZZslpLsdKYYXN70Gpu-jCLCD1yF9Y/edit?usp=sharinghere). 
 
-This course has been completely redesigning and will more than likely have its fair share of bugs. If you have anything you longed for please make an [issue](https://github.com/BCDevOps/devops-platform-workshops/issues/new/choose) on github :) 
+The lab materials have been designed so that you generally follow a happy path for a simple two-tier deployment. There will be many intentional bugs along the way that will have you encounter real-life troubleshooting techniques.
 
-## Who is this for?
+__Please be aware the throughout the labs, you will be guided to make common mistakes, and then guide you on how to fix them.__
```
Comment (ShellyXueHan) [inline 101-lab/content/00_overview.md:24]:
```suggestion
__Please be aware throughout the labs, you will be guided to make common mistakes, and then guide you on how to fix them.__
```

File: 101-lab/content/03_deployment.md [hunk 3]
```diff
@@ -248,12 +248,63 @@ brew install jq`
   # Check environment variables configuration
   oc -n [-dev] get deployment/rocketchat-[username] -o json | jq '.spec.template.spec.containers[].env'
   ```
+## Network policies (self-paced training)
+
+**Note: if you're doing the live training in the d8f105-dev and d8f105-tools namespaces, skip this step**
+In your dev namespace, create the following network policies, replacing [-dev] with the name of your dev namespace.
```
Comment (ShellyXueHan) [inline 101-lab/content/03_deployment.md:254]:
```suggestion

In your dev namespace, create the following network policies, replacing [-dev] with the name of your dev namespace.
```

**Review by ShellyXueHan — APPROVED** [review]
LGTM, just a few minor things

File: 101-lab/content/00_overview.md (hunk 1 above)
Comment (mtspn) [reply 101-lab/content/00_overview.md:20]: That's a special power for you and the rest of our training facilitators, Iryna just set it up this morning. I've just double checked from incognito and have only view access.

**Pull request merged**